olive/passes/onnx/gptq.py

```diff
@@ -0,0 +1,213 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+from typing import Any, Callable, Dict, Union
+
+import torch
+from packaging import version
+
+from olive.common.config_utils import validate_config
+from olive.data.config import DataConfig
+from olive.hardware.accelerator import AcceleratorSpec, Device
+from olive.model import PyTorchModelHandler
+from olive.model.handler.onnx import ONNXModelHandler
+from olive.passes import Pass
+from olive.passes.pass_config import PassConfigParam
+
+logger = logging.getLogger(__name__)
+
+
+class GptqQuantizer(Pass):
+    """GPTQ quantization using Hugging Face Optimum and export model with onnxruntime optimized kernel."""
+
+    _requires_user_script = True
+
+    @staticmethod
+    def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
```

Comment:
please rebase from main the signature is changed to classmethod

Reply:
Please change the method to classmethod.

Reply:
This file should be move to pytorch/gptq.py, I've changed in the new file